.github/workflows/windows-qc-dxc-d3d12.yaml

```diff
@@ -0,0 +1,21 @@
+name: Windows D3D12 AMD DXC
+
+permissions:
+  contents: read
+  checks: write
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *' # run every 30 minutes
```

Comment:
Why is this one 30 minutes and the others 2 hours?

Reply:
The comment is wrong here because this is actually every hour... but it also shouldn't really be different (at least not initially). So I'm going to change this to every 2 hours as well and we can see where the bot utilization is.